rpm/0001-Add-rpm-spec-and-.pro-files.patch

```diff
@@ -1,7 +1,7 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Bea Lam <bea.lam@jollamobile.com>
 Date: Thu, 21 Jul 2016 05:37:23 +0930
-Subject: [PATCH] [packaging] Add rpm spec and .pro files
+Subject: [PATCH] Add rpm spec and .pro files
```

Comment:
While the original commit added spec too, the patch does not -> subject could be adjusted?

Reply:
Was about to adjust manually on the patch but then realized it would the next time create a patch file with different name with git format-patch. Since we should get rid of the whole .pro files on hopefully the next update, perhaps I just just leave it as it is now.

Reply:
You can keep the tag by using git format-patch -k/--keep-subject

Reply:
" Do not strip/add [PATCH] from the first line of the commit log message." - that doesn't help here much. Point was that "rpm spec" should be removed from the subject and if I did that manually, git had no way to know to generate such words when creating patches the next time.